### PR TITLE
New page for push notification health targets

### PR DESCRIPTION
**Pull request by cwarnermm**
Incorporated [Engineering thought leadership and guidance](https://mattermost.atlassian.net/wiki/spaces/ICU/pages/3111026690/Push+Notification+Health+Targets) into the product docs.

### CODE REVIEW

File: source/scale/push-notification-health-targets.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
+Push notification health targets
+=================================
+
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+When using the `Mattermost Notification Health <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_ Grafana dashboard to track different types of notifications sent from Mattermost, we recommend adhering to the following health targets to ensure a performant production deployment of Mattermost.
```
Comment (cwarnermm) [inline source/scale/push-notification-health-targets.rst:7]:
```suggestion
When using the `Mattermost Notification Health <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_ Grafana dashboard to track different types of notifications sent from Mattermost, we recommend adhering to the following mobile push notification health targets to ensure a performant production deployment of Mattermost.
```

File: source/scale/push-notification-health-targets.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+.. note::
+
+    - Accessing and enabling Mattermost Notification Health Monitoring requires the feature flag ``NotificationMonitoring`` to be set to ``true``. 
+    - System admins can :ref:`disable notification monitoring data collection <configure/site-configuration-settings:enable notification monitoring>` through the System Console.
+
+Push Proxy Delivery Rate
+------------------------
+
+The **Push Proxy Delivery Rate** panel indicates the percentage of push notifications that have been successfully delivered to the endpoints (i.e., mobile devices) out of the total number of push notifications sent.
```
Comment (larkox) [inline source/scale/push-notification-health-targets.rst:17]: Not exactly the devices. It is more about the Push Notification Service (Apple Push Notification service (APNs) or Firebase Cloud Messaging (FCM)).

File: source/scale/push-notification-health-targets.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+This information helps admins to pinpoint common problems affecting push notification delivery, diagnose and fix issues more efficiently, detect patterns and trends, identify intermittent issues before they become more serious, and enhance the overall user experience with the Mattermost platform.
+
+Missing profile errors are generally safe to ignore. However, if you see occurences of this error increase, verify logs for ``push_proxy_send_error``.
+
+Push Notifications Not Sent Distribution
+------------------------------------------
+
+The **Push Notifications Not Sent Distribution** panel helps admins ensure the robustness and reliability of the push notification system by providing detailed insights into the specific reasons for notification delivery failures. This panel displays the distribution of push notifications that were not sent, categorized by the specific reasons why they were not sent. This information helps admins understand the causes behind undelivered push notifications.
```
Comment (larkox) [inline source/scale/push-notification-health-targets.rst:37]: Not really delivery failures. In this context, it is not a failure the notification was not sent. It is expected that the notification was not sent, and this tell us the reason why that is expected.

For example, a notification is not sent, if the channel is muted.

**Review by larkox — CHANGES_REQUESTED** [review]
Great work! Just two small things to rephrase.

**Comment by cwarnermm** [comment]
Great feedback, @larkox, thank you! I've incorporated your changes in my latest commit.

**Pull request merged**